R/summary_skew_kurt.qmd

```diff
@@ -193,4 +178,16 @@ and the `Kurtosis()` function a Type 2 Kurtosis.
 
 ```
 
+## Reference
+
+::: {.callout-note collapse="true" title="Session Info"}
+```{r}
+#| echo: false
 
+# List all the packages needed 
+si <- sessioninfo::session_info(c(
+  #Create a vector of all the packages used in this file 
```

Comment:
```suggestion
  "e1071","procs","sasLM","moments"
```

Reply:
Thank you for adding this, we just want to make sure to add the packages used in the document to the session-info